ipatool

```diff
@@ -166,6 +169,7 @@ import pprint
 
 import yaml       # yum install python3-PyYAML
 import blessings  # yum install python3-blessings
+import bugzilla   # dnf isntall python3-bugzilla
```

Comment:
nit: typo isntall -> install

should all imports examples switch from yum to dnf for consistency?

Reply:
More than that, I've fixed the order and moved to the top so the linter would stop complaining.